src/components/Game.tsx

```diff
@@ -3,19 +3,41 @@ import React from "react";
 import GuessList from "./GuessList";
 import { Guess } from "../model/guess";
 import { NUM_OF_GUESSES_ALLOWED } from "../constants";
+import { words } from "../data";
+import { checkGuess, pickAnswer } from "../utils";
+import { Status } from "../model/status";
 
 function Game() {
+  const [answer, setAnswer] = React.useState<string>(() => pickAnswer(words));
   const [currentGuesses, setCurrentGuesses] = React.useState<Guess[]>([]);
+  const [isPlaying, setIsPlaying] = React.useState<boolean>(true);
```

Comment:
minor suggestion, entrando nel mondo delle opinioni personali:
```suggestion
  const [isPlaying, setIsPlaying] = React.useState(true);
```

Non è richiesto passare il tipo generic quando può essere inferito automaticamente. Nessuno ovviamente ti vieta di farlo, ma alla lunga diminuisci il boilerplate e hai un codice molto più leggibile.

Se ci aggiungi anche cherry-picking dell'import:
```
import { useState } from 'react';
```

ti ritrovi con un codice molto più snello:

```
const [isPlaying, setIsPlaying] = useState(true)
```